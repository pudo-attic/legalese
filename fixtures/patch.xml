
ops: create | delete | replace 
convenience_ops: copy


changeset = 

<patch document="hasen">
  <create to="/4/g/37/n">
    
    <p>I exist.</p>
  </create>
</patch>

<patch document="stgb@47587">
    <append to="§5">
        <p>Ich bin ein hase</p>
    </append>
</patch>
        
    
