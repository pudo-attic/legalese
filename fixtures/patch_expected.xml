<document>
  sun
  a<one/>b<two/>c
  <section ref="p1">
    1<a />2<b />3
    g<five/>h<six/>i
  </section>
  moon
  <section ref="p2">
    j<seven/>k<eight/>l
    4<c />5<d />6
  </section>
  d<three/>e<four/>f
  earth
</document>
